CHANGELOG.md

```diff
@@ -48,6 +48,13 @@ _None._
 
 _None._
 
+## 2.18.1
+
+### Bug Fixes
+
+- `annotate_test_failures` no longer prints an empty `<code>` box when reporting a `failure` with no extra `details`. [#63]
```

Comment:
Whoops on the duplicate entry from a copy/paste 🤦  [Fixed directly on `trunk`](d3799444517fbaa86415bd092ca449574e453302).